tools/tpm2_ptool.py

```diff
@@ -924,6 +925,10 @@ def generate_options(self, group_parser):
             help='The type of the key.\n',
             choices=[ 'rsa1024', 'rsa2048', 'aes128', 'aes256' ],
             required=True)
+        group_parser.add_argument(
+            '--keylabel',
+            help='The key label to identify the key. Defaults to testkey\n',
+            default="testkey")
```

Comment:
Then you'll have duplicate labels. Why not use the primary key integer from the db as a string?

Reply:
perhaps output the label in addkey, which if you use the primary key for the object in the db, is already correct. So it just needs minor logic tweak.